pandas/core/window/ewm.py

```diff
@@ -919,6 +919,12 @@ def _get_window_indexer(self) -> GroupbyIndexer:
         )
         return window_indexer
 
+    @doc(_shared_docs["aggregate"])
```

Comment:
Shouldn't this and below be inherited from the base class?

Reply:
I think so. This was an ugly workaround to a mypy error I was getting but using `type: ignore[misc]` instead now